CONTRIBUTING.md

```diff
@@ -4,13 +4,13 @@ If you have a usage question for a product built on MapLibre GL (such as our iOS
 
 If you want to contribute code:
 
-1. Please familiarize yourself with the [install process](INSTALL.md).
+1. Please familiarize yourself with the [install process](README.md#installation).
 
-1. Ensure that existing [pull requests](https://github.com/mapbox/mapbox-gl-native-ios/pulls) and [issues](https://github.com/mapbox/mapbox-gl-native-ios/issues) don’t already cover your contribution or question.
+1. Ensure that existing [pull requests](https://github.com/maplibre/maplibre-gl-native/pulls) and [issues](https://github.com/maplibre/maplibre-gl-native/issues) don’t already cover your contribution or question.
 
 1. Pull requests are gladly accepted. If there are any changes that developers using one of the GL SDKs should be aware of, please update the **master** section of the relevant changelog(s):
-  * [Mapbox Maps SDK for iOS](platform/ios/CHANGELOG.md)
-  * [Mapbox Maps SDK for macOS](platform/macos/CHANGELOG.md)
+  * [Maplibre Maps SDK for iOS](platform/ios/CHANGELOG.md)
```

Comment:
The name of the sdk is still mapbox

Reply:
I do not understand, the changes seem to be in line with #297. That is after #297 https://github.com/maplibre/maplibre-gl-native/tree/main/platform/darwin already uses the same anchors for the same files as CONTRIBUTING.md would after this pull request. So as I see it either #297 is wrong or this pull request is correct.

Reply:
You are right @xabbu42. I think we should just move forward and replace Mapbox with MapLibre...